### PR TITLE
Study manager JavaDoc and UserGuide documentation

### PR DESCRIPTION
**Pull request by mathanmahe**

### CODE REVIEW

**Review by YitHien — COMMENTED** [review]
JavaDoc and UserGuide format looks good!

File: src/main/java/seedu/allonus/modules/Module.java [hunk 1]
```diff
@@ -1,11 +1,23 @@
 package seedu.allonus.modules;
 
+/**
+ * Represents a module at NUS with attributes module code, module category, day of the week, and timeslot of class.
+ */
```
Comment (OzairHasan) [inline src/main/java/seedu/allonus/modules/Module.java:5]: I really like the use of javadoc headers throughout the files. Very helpful.

**Review by OzairHasan — COMMENTED** [review]
LGTM

**Comment by mathanmahe** [comment]
This fixes the extra warning log that was shown during testing.

**Pull request merged**